src/components/Card/FAQcard.css

```diff
@@ -0,0 +1,27 @@
+.lil-faqcard {
+  /*
+  background-color needs to be updated
+  */
+  height: 280px;
+  width: 252px;
```

Comment:
responsive ?

Reply:
Yes

Reply:
lets make width `250px` then...we shud not use random values

Reply:
okay will make it 250, I used the values given in figma